.env-sample

```diff
@@ -0,0 +1,5 @@
+BASIC_AUTH_ZEN=ommm:$apr1$1n0zqO2y$A2ueB9zMWZTYTtfckFhql1
+TRAEFIK_HOST_ZEN=myhost.com
```

Comment:
Could you use `example.com` instead? Same below. I think it's a better candidate.